main.go

```diff
@@ -1,7 +1,99 @@
 package main
 
-import "fmt"
+import (
+	"./conf"
+	"./models"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/labstack/echo/v4"
+	"github.com/tkanos/gonfig"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+	"log"
+	"net/http"
+	"strconv"
+)
 
 func main() {
-	fmt.Println("18.20 at Sep 07 2016 7:20 IR721")
+	configuration := conf.Configuration{}
+	_ = gonfig.GetConf("conf.json", &configuration)
+
+	ctx := context.Background() ///WithTimeout(context.Background(), 10*time.Second)
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI("mongodb://localhost:27017"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = client.Ping(ctx, readpref.Primary())
```

Comment:
I did not see this earlier. can you explain it to me?

Reply:
Ping verifies that the client can connect to the topology. If readPreference is nil then will use the client's default read preference.
https://godoc.org/go.mongodb.org/mongo-driver/mongo#Client.Ping